plugins/Scripts/Plugins/Registration/Correct_3D_drift.py

```diff
@@ -22,16 +22,42 @@
 from ij.gui import GenericDialog, YesNoCancelDialog, Roi
 from mpicbg.imglib.image import ImagePlusAdapter
 from mpicbg.imglib.algorithm.fft import PhaseCorrelation
-from org.scijava.vecmath import Point3i
-from org.scijava.vecmath import Point3f
+from javax.vecmath import Point3i
+from javax.vecmath import Point3f
```

Comment:
The recommended way is to modify this very pull request by rebasing and force-pushing to the topic branch (`tischi:patch-3` in your case). See the [documentation on the wiki](http://imagej.net/How_to_contribute_to_an_existing_plugin_or_library#Modifying_your_commits).